src/Console/Command/Init.php

```diff
@@ -6,20 +6,23 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
-use Symfony\Component\Filesystem\Filesystem;
 use Symfony\Component\Process\ExecutableFinder;
 use Symfony\Component\Process\Process;
 
 final class Init extends Command
 {
+    private SymfonyStyle $io;
+    private string $rootPath;
```

Comment:
Unfortunately this isn't compatible with PHP 7.3

Reply:
This should do it!
```
/**
     * Symfony console input/output handler
     *
     * @var SymfonyStyle
     */
    private $io;

    /**
     * Absolute path pointing into project's
     * template directory
     *
     * @var string
     */
    private $rootPath;
```